airbyte-integrations/connectors/source-stripe/source_stripe/source.py

```diff
@@ -524,7 +524,13 @@ def streams(self, config: MutableMapping[str, Any]) -> List[Stream]:
 
     def _to_concurrent(self, stream: Stream, fallback_start, state_manager: ConnectorStateManager) -> Stream:
         if stream.name in self._streams_configured_as_full_refresh:
-            return StreamFacade.create_from_stream(stream, self, entrypoint_logger, self._create_empty_state(), NoopCursor())
+            return StreamFacade.create_from_stream(
```

Comment:
just for pre-release testing

Reply:
Looks like this makes `source-stripe` to emit record counts by virtue of emitting final state on full refresh? Am I reading this right?

Reply:
If so, perhaps worth a unit test?

Reply:
this is being removed because this is solely to test, I'm very much against trying to fix sources in the same PR as a CDK change because the source itself isn't actually using the published CDK version since it hasn't been merged yet

That being said, for the future stripe change, probably more of a mock server test rather than a unit test, because we might not have a flow that tests this e2e as a unit test